test/DriverFactoryTest.php

```diff
@@ -12,39 +12,37 @@
 use Roave\PsrContainerDoctrine\DriverFactory;
 use Roave\PsrContainerDoctrine\Exception\OutOfBoundsException;
 
-class DriverFactoryTest extends TestCase
+final class DriverFactoryTest extends TestCase
 {
     public function testMissingClassKeyWillReturnOutOfBoundException() : void
     {
-        $container = $this->prophesize(ContainerInterface::class);
+        $container = $this->getMockBuilder(ContainerInterface::class)->getMock();
```

Comment:
`$this->createMock()` suffices

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2